app/Http/Controllers/Localization.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Session;
+
+class Localization extends Controller
+{
+    public function changeLanguage(Request $request)
+    {
+        $lang = $request->language;
+        $language = config('app.locale');
+        if ($lang == 'en') {
+            $language = 'en';
+        }
+        if ($lang == 'vi') {
+            $language = 'vi';
+        }
+        Session::put('language', $lang);
```

Comment:
```suggestion
        $lang = $request->language;
        Session::put('language', $lang);
```
đoạn if m thấy ko cần thiết bỏ đi nha